tools/windows/mgba-setup-win32.iss

```diff
@@ -0,0 +1,182 @@
+;For automation purposes it is highly recommended to copy the files from
+;\tools\windows\ to the directory that contains the win32 distribution files!
+
+;Set CurrentReleaseVersion to the number of the latest stable mGBA build.
+#define CurrentReleaseVersion = '0.6.3'

```

Comment:
It might make sense to make this a .in file that's processed by CMake so it can get the current version number from the version scripts--unless there's a way to run a command in the .iss folder to get the version from CMake.